package.json

```diff
@@ -113,7 +113,7 @@
     "mediaelement": "4.2.15",
     "mediaelement-plugins": "2.5.1",
     "openseadragon": "2.4.2",
-    "pdfjs-dist": "4.6.82",
+    "pdfjs-dist": "2.0.161",
```

Comment:
Do you think it's worth changing this to 3.11.174 while we're changing it again anyway, so it at least reflects the version that is actually being used? That way, if we get any security alerts, we'll actually know they're for real. :-)